src/components/auth/landing.js

```diff
@@ -0,0 +1,321 @@
+        </section>
+
+        <section class="section-five">
+          <div class="container">
+            <h2>CLIENT TESTIMONIALS</h2>
+            <p>Check out some students reviews and their experience</p>
+            <p>Real Students, Real reviews</p>
+          </div>
+
+          <div class="container testimonies">
+            <div class="testimony testimony-1">
+              <div>
+                <div class="student-image"></div>
+                <h2 class="text-sm">Lorem ipsum</h2>
+              </div>
+
+              <p>
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Culpa
+                quas molestias maxime earum
+              </p>
+            </div>
+
+            <div class="testimony testimony-2">
+              <div>
+                <div class="student-image"></div>
+                <h2 class="text-sm">Lorem ipsum</h2>
+              </div>
+
+              <p>
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Culpa
+                quas molestias maxime earum
+              </p>
+            </div>
+
+            <div class="testimony testimony-3">
+              <div>
+                <div class="student-image"></div>
+                <h2 class="text-sm">Lorem ipsum</h2>
+              </div>
+
+              <p>
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Culpa
+                quas molestias maxime earum
+              </p>
+            </div>
+          </div>
+
+          <div class="text-center d-flex flex-column justify-content-center">
+            <h2>Still have more questions?</h2>
+            <p>
+              Lorem ipsum, dolor sit amet consectetur adipisicing elit.
+              Voluptatem accusantium ea cupiditate architecto, quasi
+              praesentium!
+            </p>
+            <a href="" class="btn btn-md text-white bg-blue">
+              Get Started
+            </a>
+          </div>
+        </section>
+
+        <section class="section-six">
+          <h2 class="container text-white footer-text-bold">CodeClanNigera</h2>
+          <div class="container footer-links-wrap">
+            <ul>
+              <li class="footer-text-bold">SERVICES</li>
+              <li>
+                <a href="#">Blog</a>
+              </li>
+              <li>
+                <a href="#">Community</a>
+              </li>
+              <li>
+                <a href="#">Resources</a>
+              </li>
+            </ul>
+
+            <ul>
+              <li class="footer-text-bold">ABOUT US</li>
+              <li>
+                <a href="#">Mentors</a>
+              </li>
+              <li>
+                <a href="#">Partners</a>
+              </li>
+              <li>
+                <a href="#">Newsletter</a>
+              </li>
+            </ul>
+
+            <ul>
+              <li class="footer-text-bold">CONNECT WITH US</li>
+              <li>
+                <a href="#">Facebook</a>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The href attribute requires a valid value to be accessible. Provide a valid, navigable address as the href value. If you cannot provide a valid href, but still need the element to resemble a link, use a button and change it with appropriate styles. Learn more: https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-frontend/pullRequest?prid=5641179)